### ISSUE TITLE
Setting admin password at install time does not work

### ISSUE THREAD

**Issue by fallen** [YunoHost-Apps/pytition_ynh#6]
In fact the `DJANGO_SUPERUSER_PASSWORD` environment variable is only working from Django 3.0 onward.
Pytition is using Django 2.2.

Only solution I can see would be to use the interactive version of `manage.py createsuperuser` with some bash wizardry or some expect script?

**Comment by Salamandar** [comment]
Aaaaah thanks. I was wondering why that was not working.
Yeah, I'll fix that.

**Comment by Salamandar** [comment]
Crap, gotta do that. 

In the meantime, a workaround:
```
# Check which password was saved by Yunohost
cat /etc/yunohost/apps/pytition/settings.yml

# Should be this, but if you have multiple instances that might be something else
cd /var/www/pytition

# Source the python environment
source venv/bin/activate

cd pytition/pytition
export DJANGO_SETTINGS_MODULE=pytition.settings.config

# It will prompt you a new password (twice)
python3 [manage.py](http://manage.py/) changepassword admin
```

**Comment by fallen** [comment]
I published a new release of Pytition [v2.8](https://github.com/pytition/Pytition/releases/tag/v2.8) base on Django 4.2 (current LTS).
This should improve the situation because it allows `DJANGO_SUPERUSER_PASSWORD` to be used :)

**Comment by Salamandar** [comment]
Fixed by #24 and #28 !
